readinglist/backend/__init__.py

```diff
@@ -36,6 +36,16 @@ def get_all(self, resource, user_id, filters=None, sorting=None,
                 pagination_rules=None, limit=None):
         raise NotImplementedError
 
+    def strip_deleted_record(self, resource, user_id, record):
+        """Strip the record to its deleted version (in place)."""
```

Comment:
Yes, it strips (in place) all the fields that aren't id or last-modified. And then it has the deleted mark defined at the resource level